src/pages/ESNMembersTab/EsnersSheet.tsx

```diff
@@ -43,15 +62,20 @@ function EsnersSheet() {
 
     if (esners === []) return <div> NO ESNERS FOUND </div>;
     return (
-        <div>
-            {esners.map((data: ESNer) => (
-                <span key={data.id}>
-                    <p>
-                        {data.name} {data.surname} <strong>{data.position}</strong>
-                    </p>
-                </span>
-            ))}
-        </div>
+        <IonList>
+            <IonItemGroup>
+                <IonLabel>Board</IonLabel>
+                {esners.filter(member => member.memberType === MEMBER_BOARD).map((data: ESNer) => showMembers(data))}
```

Comment:
Please don't put any logic into `render` function.

You can make this filtering before, assign the result to const and in render place those const only.

```const boardMembers = esners.filter(member => member.memberType === MEMBER_BOARD).map((data: ESNer) => showMembers(data)) ```

 ```
 <IonItemGroup>
<IonLabel>Board</IonLabel>
{boardMembers}
</IonItemGroup>
```

Reply:
I guess you mean `return` not `render`?
Mapping was already in this function before, so I added filtering in the same place.

Reply:
Yeah (I need a ☕...), but in fact this all is a render function and in this 'return' shows which component will be rendered :P 

Nevertheless, it's good practice to keep logic away from this place. Mapping only is acceptable because in fact you are only rendering 'some' number of same components. 
While filtering you need to apply some logic. 

Second thougth. I'd do here filtering in code and assign result into const, and then in render do `boardMembers.map((data: ESNer) => showMembers(data))`. This solution allows to clearly show that we want to render list of items

Reply:
Ok, I made the corrections. Check and close ;)

Reply:
I did like you suggested in the first comment, and I'd leave it like this.